architecture/infrastructure_components/web_console.adoc

```diff
@@ -49,6 +51,20 @@ link:../../install_config/master_node_configuration.html#master-configuration-fi
 configuration file parameter `corsAllowedOrigins`].
 endif::[]
 
+ifdef::openshift-origin,openshift-online,openshift-enterprise,openshift-dedicated[]
+[[web-console-cli-downloads]]
+
+== CLI Downloads
+You can download and unpack the CLI from the
+link:../install_config/web_console_customization.html#adding-or-changing-links-to-download-the-cli[*About*
+page on the web console] for use on Linux, MacOSX, and Windows clients
+ifdef::openshift-enterprise,openshift-origin[]
+if your cluster administrator has enabled it
+endif::[]
+:
+
+image::about_page.png["About Page"]
+
```

Comment:
@bfallonf @vikram-redhat @ahardin-rh This should not get published into the OSE 3.1 docs yet. It's a 3.2 feature. I'm guessing this got pulled in by another commit or introduced maybe while resolving a merge conflict.